pkg/storegateway/indexheader/index/symbols_test.go

```diff
@@ -49,7 +49,7 @@ func TestSymbols(t *testing.T) {
 	require.NoError(t, err)
 
 	// We store only 4 offsets to symbols.
-	require.Equal(t, 32, s.Size())
+	require.Equal(t, 4, len(s.offsets))
```

Comment:
Nit, but simpler and more intentional :)
```suggestion
	require.Len(t, s.offsets, 4)
```